test/parallel/test-assert-fail.js

```diff
@@ -23,7 +23,7 @@ assert.throws(
     code: 'ERR_ASSERTION',
     type: assert.AssertionError,
     operator: undefined,
-    actual: undefined,
+    actual: 'custom message',
```

Comment:
It is not accompanied by any changes to `lib` or `src`, so it really can’t be. I have to admit I didn’t really figure out why this is required, if you like, feel free to do the digging.